### ISSUE TITLE
Grouped Components - Widgets for default values

### ISSUE THREAD

**Issue by farmaazon** [enso-org/enso#11246]
To be done after #11408 and #8754

The simple input field for default argument should be replaced with actual subwidget, and special drop-down to choose required/default options. The "normal" set of widgets should be matchable basing on the current argument's type and value if any. 

If there is no default value, we should match the same way as for placeholders, falling back to Code Widget if no match.

In general, this task's focus is on simple literals; the default values being function calls are not in scope.

**Comment by farmaazon** [comment]
Refinement notes:
* Special widget input for argument + special widget displaying arg, type and placeholder for default value
* the default arg widget may accept only single line code.
* Any ports inside cannot be connectable.
* Any problematic case should be just disabled and moved to a separate issue.

**Comment by farmaazon** [comment]
Refinement notes, vol. 2:
* The selection of require/default is very unusual, unless we remove “default” from the UI - then it’s just a selection with additional “required” option (and default at least if there is no other option)
